src/Nethermind/Nethermind.Hive/HivePlugin.cs

```diff
@@ -46,23 +46,11 @@ public Task InitNetworkProtocol()
             if (Enabled)
             {
                 if (_api.SyncPeerPool == null) throw new ArgumentNullException(nameof(_api.SyncPeerPool));
-
-                _api.SyncPeerPool.PeerRefreshed += OnPeerRefreshed;
             }
 
             return Task.CompletedTask;
         }
 
-        private void OnPeerRefreshed(object? sender, PeerHeadRefreshedEventArgs e)
-        {
```

Comment:
Consult it with @marcindsobczak. I think it was needed for some other hive test simulator

Reply:
It was added for eth protocol test `TestBlockHashAnnounce`, right now we are failing eth protocol tests anyway, because we don't have eth67, and we can't add eth67 because it is dropping GetNodeData and we will not be able to sync anymore. So I think at this point feel free to remove it, in the future when fixing remaining failing tests we will figure out another solution